src/terminal/init.lua

```diff
@@ -127,21 +119,19 @@ function terminal.sleep(is_blocking)
   end
 end
 
-
 local M = {
-  _VERSION = "0.0.1",
-  _COPYRIGHT = "Copyright (c) 2024-2024 Thijs Schreijer",
-  _DESCRIPTION = "Cross platform terminal library for Lua (Windows/Unix/Mac)",
+    _VERSION = "0.0.1",
+    _COPYRIGHT = "Copyright (c) 2024-2024 Thijs Schreijer",
+    _DESCRIPTION = "Cross platform terminal library for Lua (Windows/Unix/Mac)",
```

Comment:
Can we undo the whitespace changes? it should be a 2 space indent (see file `.editorconfig`)

Reply:
Also good practice is to keep commits clean and atomic. So make 1 change per commit. Don't mix multiple features, fixes, or anything else in a single commit.
I just added the `CONTRIBUTING.md` file today, might be worth while to read that.